tt-train/sources/ttml/modules/llama_block.hpp

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: (c) 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "autograd/tensor.hpp"
+#include "modules/dropout_module.hpp"
+#include "modules/linear_module.hpp"
+#include "modules/multi_head_attention.hpp"
+#include "modules/rms_norm_module.hpp"
+#include "modules/single_head_attention.hpp"
+
+namespace ttml::modules {
+
+class LlamaMLP : public autograd::ModuleBase {
+    std::shared_ptr<LinearLayer> w1;
```

Comment:
lets start all members with m_. also I like  m_linear1 and etc. but w1 w2 are used in priginal implementation so could be ok.